cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -66,17 +68,16 @@ def test_service_start(appliance, setup_provider, context,
     catalog_item = order_service
     with appliance.context.use(context):
         my_service = MyService(appliance, catalog_item.name)
-        if provider.one_of(InfraProvider):
+        if provider.one_of(InfraProvider, EC2Provider, AzureProvider):
             # For Infra providers vm is provisioned.Hence Stop option is shown
+            # For Azure, EC2 Providers Instance is provisioned.Hence Stop option is shown
             my_service.service_power(power='Stop')
             view = my_service.create_view(DetailsMyServiceView)
-            view.notification.assert_message(
-                "{} was {}.".format(catalog_item.name, 'stopped'))
+            assert view.power_operations.item_enabled('Start')
```

Comment:
Question: Can be this button enabled when the process of stopping is not finished?

Reply:
@rlbabyuk  yes, start button gets enabled while the stopping process is not yet finished.

Reply:
There's a TODO in the `service_power` method to verify the VM states change using the REST API. That will be the ideal way to verify that the service was actually stopped. But I am curious, why wasn't `view.notification.assert_message` working?

Reply:
I'll write test using API later. I have written it using UI itself. it was long pending on me.